modyn/selector/internal/selector_strategies/downsampling_strategies/downsampling_scheduler.py

```diff
@@ -71,7 +73,7 @@ def training_status_bar_scale(self) -> int:
         return self.current_downsampler.status_bar_scale
 
     def inform_next_trigger(self, next_trigger_id: int, selector_storage_backend: AbstractStorageBackend) -> None:
-        if self.next_threshold is not None and next_trigger_id >= self.next_threshold:
+        if self.next_threshold is not None and next_trigger_id - self._warmup_triggers >= self.next_threshold:
```

Comment:
can you add a comment in Pydantic config how `warmup_triggers` will affect the calculation of `downsampling_thresholds`?

Reply:
Will do (but note it does not affect it. This logic takes care that you don't need to think about it)

Reply:
Yes. but how the threshold takes care of now is *after* the warm up period.

Reply:
Yes, exactly, but you don't need to adjust the indices or something. That's what I mean. Anyways, I hope the config comment is sufficient